vernac/declare.ml

```diff
@@ -469,7 +482,17 @@ let declare_constant ?(local = Locality.ImportDefaultBehavior) ~name ~kind ~typi
     | Def _ | Undef _ | Primitive _ | Symbol _ -> assert false
   in
   let () = register_constant kn kind local ?user_warns in
-  kn
+  let univs = match cd with
+    | DefinitionEntry { proof_entry_universes = univs }
+    | ParameterEntry { parameter_entry_universes = univs }
+    | SymbolEntry { symb_entry_universes = univs } ->
+      fst univs
+  in
+  let univs = match univs with
+    | Monomorphic_entry _ -> UVars.Instance.empty
+    | Polymorphic_entry uctx -> UVars.UContext.instance uctx
```

Comment:
These lines of code are used at least 2 other times in the code. Would it make sense to add them to the universe API (in particular, they seem obvious to expert, but documenting why they are canonical in some sense - if so - would help non-expert like me).